src/SDKs/SqlManagement/Management.Sql/project.json

```diff
@@ -1,5 +1,5 @@
 ﻿{
-  "version": "1.2.0-preview",
+  "version": "1.3.0-preview",
```

Comment:
@jaredmoo post migration, we no longer need project.json, global.json and *.xproj.
These might have been left around if I found it will be useful to reference what package references were used earlier prior to migration.
If everything looks good, you can delete these files.